### ISSUE TITLE
Is there a way to print browser network logs during execution?

### ISSUE THREAD

**Issue by nipun0212** [SAP/ui5-uiveri5#274]

**Comment by maximnaidenov** [comment]
No, not for now. We have plans to gather and download a HAR file for the network stack but have not reached there. Anaway, you can experiment with the browser.driver.manage().logs() API and try to get at least some info.
